javascripts/content_scripts/posting.controller.js

```diff
@@ -0,0 +1,280 @@
+/**
+ * @fileOverview Control the seamless-posting form:
+ * respond to the click of the Privly button.
+ *
```

Comment:
It isn't clear what is the responsibility of the seamless app in privly-applications, and what is the responsibility of privly-chrome to support that app. You need commenting to separate the two.